src/institutions/Institutions.css

```diff
@@ -1,10 +1,15 @@
 .Institutions {
-  max-width: 800px;
+  margin-top: 3em;
+}
+
+.Institutions > * {
```

Comment:
Here we just add the `max-width` to all the children directly.

Reply:
fwiw, probably the quicker solution is to just add the relevant grid styling, since we're still including the `usa-grid` class

Reply:
I'm not sure that's the case throughout the app, but that's what I'll try to do tomorrow first thing.